src/main/java/ro/unibuc/hello/validation/ValueOfEnumValidator.java

```diff
@@ -0,0 +1,26 @@
+package ro.unibuc.hello.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class ValueOfEnumValidator implements ConstraintValidator<ValueOfEnum, String> {
```

Comment:
It's so strange that there are no pre-implemented helpers for such a common functionality.

Reply:
I agree :)))